### DISCUSSION TITLE
Battery drain on Kobo Sage

### DISCUSSION THREAD

**Discussion by sokbear** [koreader/koreader#11306] [Q&A]
Hello! I have Koreader 2023.10 installed on Kobo Sage. It's beautiful programm but a battery usage could be up to 20% in hour. I made a little investigation and have discovered that the continues current consumption when Koreader running with light and WiFi off is about 120 - 130 mA. On the stock programm this value about zero mA. Is there a way to reduce a current consumption?

**Comment by NiLuJe** [comment]
Enable autostandby in the Device settings.

Also, I have no idea how you're seeing a near zero amp draw in standby, because the numbers are so very very far from that: https://github.com/koreader/koreader/issues/9435#issuecomment-1214257679 (that's an Elipsa, but we have Sage numbers... somewhere, and they're not so far off).

EDIT: Oh, lights off. Never checked with lights off, but it's a fairly large part of the power budget, so, that makes slightly more sense ;). Still far from zero, though ;).

**Comment by sokbear** [comment]
Thank you for advice. It's really help.
